social_core/backends/vk.py

```diff
@@ -178,7 +178,7 @@ def vk_api(backend, method, data):
         http://goo.gl/yLcaa
     """
     # We need to perform server-side call if no access_token
-    data['v'] = backend.setting('API_VERSION', '5.131')
+    data['v'] = backend.setting('API_VERSION', '5.81')
```

Comment:
You are changing this to *older* version. See also https://github.com/python-social-auth/social-core/pull/613 or https://github.com/python-social-auth/social-core/pull/615